app/Main.hs

```diff
@@ -4,11 +4,11 @@ module Main where
 
 import Data.Text (Text)
 import Tablebot (BotConfig (..), runTablebotWithEnv)
-import Tablebot.Plugins (allPlugins)
+import Tablebot.Plugins (allPlugins, minusPl)
 
 -- @main@ runs forever. This allows bot reloading by fully shutting down the bot and letting it restart.
 main :: IO ()
-main = runTablebotWithEnv allPlugins $ BotConfig {gamePlaying = "with dice", rootHelpText = rootBody}
+main = runTablebotWithEnv (allPlugins `minusPl` ["ping"]) $ BotConfig {gamePlaying = "with dice", rootHelpText = rootBody}
```

Comment:
OOI, why are we doing this with an extra command instead of just commenting it out of the list? This seems un-neccicarialy confusing for users who might want to actually have ping enabled?

Reply:
That does seem confusing, I agree. I'll revert that change. I'm also keeping ping, so I'll just remove pong from the ping plugin.